docker-compose.yml

```diff
@@ -1,10 +1,12 @@
 version: '3'
 services:
   gpt-researcher:
-    image: assafelovic/gpt-researcher
+    image: kramer1346/gpt-researcher
     build: ./
     environment: 
       OPENAI_API_KEY: ${OPENAI_API_KEY}
       TAVILY_API_KEY: ${TAVILY_API_KEY}
+      DOC_PATH: "=./docs/my-docs"
```

Comment:
@ElishaKay are you sure we should keep this static here? Probably keep it null as default?